### PR TITLE
test: tidy up signal signal input/query/model signature tests

### PR DESCRIPTION
**Pull request by cexbrayat**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the new behavior?

This updates some tests to use the public imports from `@angular/core` now that they are available, and cleans up useless imports and inaccurate names.

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/core/test/authoring/signal_queries_signature_test.ts [hunk 1]
```diff
@@ -7,8 +7,7 @@
  */
 
 import {ElementRef, forwardRef, InjectionToken, Signal} from '@angular/core';
-// TODO: export authoring functions
-import {contentChild, contentChildren, viewChild, viewChildren} from '@angular/core/src/authoring/queries';
+import {contentChild, contentChildren, viewChild, viewChildren} from '@angular/core';
```
Comment (devversion) [inline packages/core/test/authoring/signal_queries_signature_test.ts:10]: I think we can combine the imports here, and extract `Signal` into its own with a comment like in `signal_input_signature_test`.

**Review by devversion — APPROVED** [review]

File: packages/core/test/authoring/signal_queries_signature_test.ts (hunk 1 above)
Comment (devversion) [reply packages/core/test/authoring/signal_queries_signature_test.ts:10]: Also, worth considering naming the commit: `test: XX`

Comment (cexbrayat) [reply packages/core/test/authoring/signal_queries_signature_test.ts:10]: 👍 I'll combine the imports. As `Signal` is directly used, I don't think we need to import it separately with a comment.

Comment (cexbrayat) [reply packages/core/test/authoring/signal_queries_signature_test.ts]: I renamed the commit to `test` and I added a `// tslint:disable-next-line no-duplicate-imports` for the imports that are with the special comment to make the linter happy.

**Review by devversion — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 8dea3b58880d7966ec3b6b6f424173ee3fee2142.

**Pull request closed without merging**